ref/general/csec-LDAP-registry.adoc

```diff
@@ -16,67 +16,40 @@
-
-You can configure Open Liberty to retrieve security information from an LDAP server if you are building an application that uses a preexisting user registry. LDAP is the industry standard for directory servers, providing a means to create, modify, search, and delete users and groups in your applications. There are many LDAP servers available, such as the link:https://www.ibm.com/support/knowledgecenter/SSVJJU_6.4.0/com.ibm.IBMDS.doc_6.4/c_po_SDS_overview.html[IBM Security Directory Server] and the Microsoft Active Directory. There are also open-source LDAP servers, such as link:http://www.openldap.org/[OpenLDAP] and link:http://directory.apache.org/apacheds/[ApacheDS].
-
-== LDAP registry configuration
-
-To use an existing LDAP server as a user registry, first add the link:/docs/ref/feature/#appSecurity-3.0.html[Application Security] and link:/docs/ref/feature/#ldapRegistry-3.0.html[LDAP User Registry] features to your `server.xml` file. If you need to communicate with a TSL-enabled LDAP server, you must also add the link:/docs/ref/feature/#transportSecurity-1.0.html[Transport Security] feature.
-
-Configure the LDAP user registry by adding the `<ldapRegistry>` element to your `server.xml` file. Then, specify the port number, the host DNS name or IP address, LDAP type, and other attributes, as required by your security configuration. The following example shows a simple configuration for using the IBM Security Directory server as an LDAP registry:
-
-[source,java]
-----
-
-<ldapRegistry id="LDAP" realm="SampleLdapIDSRealm"
-      host="myldap.ibm.com" port="389" ignoreCase="true"
-      baseDN="o=ibm,c=us"
-      ldapType="IBM Security Directory Server" searchTimeout="8m">
-</ldapRegistry>
-----
-For a full list of configuration elements and attributes, see the link:/docs/ref/config/#ldapRegistry.html[LDAP User Registry configuration].
+LDAP is an open protocol that supports authentication and authorization services.
+LDAP specifies the language that application servers use to communicate with the user registries that store security information such as usernames, passwords, and user groups.
+Although a link:/docs/ref/general/#csec-basic-registry.html[basic user registry] might be sufficient during development and testing of an application, an LDAP user registry provides the extra security that is required for applications in production.
 
+You can configure Open Liberty to retrieve security information from an LDAP server if you are building an application that uses a preexisting user registry.
+LDAP is the industry standard for directory servers, providing a means to create, modify, search, and delete users and groups in your applications.
```

Comment:
I think to use a more active voice here you could change "industry standard for directory servers, providing a means" to "industry standard for directory servers that provides a means".